<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryEditorWindow</class>
 <widget class="QMainWindow" name="QueryEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="cmbDatabase"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabQuery">
       <attribute name="title">
        <string>Query editor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="LQueryEditor" name="teQueryEditor">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Put your SQL here!</string>
           </property>
          </widget>
          <widget class="QueryParamsForm" name="paramsForm" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabResult">
       <attribute name="title">
        <string>Result set</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tvResultSet">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="aCommit"/>
   <addaction name="aRollback"/>
   <addaction name="aExecuteQuery"/>
   <addaction name="aExecScript"/>
   <addaction name="separator"/>
   <addaction name="aQueryHistory"/>
   <addaction name="aUpdateParams"/>
  </widget>
  <action name="aCommit">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/commit.png</normaloff>:/icons/commit.png</iconset>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
   <property name="toolTip">
    <string>Commit</string>
   </property>
  </action>
  <action name="aRollback">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/rollback.png</normaloff>:/icons/rollback.png</iconset>
   </property>
   <property name="text">
    <string>Rollback</string>
   </property>
   <property name="toolTip">
    <string>Rollback</string>
   </property>
  </action>
  <action name="aExecuteQuery">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/run.png</normaloff>:/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Execute query</string>
   </property>
   <property name="toolTip">
    <string>Execute query</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="aExecScript">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/run_script.png</normaloff>:/icons/run_script.png</iconset>
   </property>
   <property name="text">
    <string>Run SQL script</string>
   </property>
   <property name="toolTip">
    <string>Run SQL script</string>
   </property>
  </action>
  <action name="aQueryHistory">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/history.png</normaloff>:/icons/history.png</iconset>
   </property>
   <property name="text">
    <string>Query history</string>
   </property>
  </action>
  <action name="aUpdateParams">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Update params</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LQueryEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>widgets/lqueryeditor.h</header>
  </customwidget>
  <customwidget>
   <class>QueryParamsForm</class>
   <extends>QWidget</extends>
   <header>forms/queryparamsform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
  <include location="../icons.qrc"/>
  <include location="../icons.qrc"/>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
